app-crypt/qdigidoc/qdigidoc-3.12.2.ebuild

```diff
@@ -19,7 +19,7 @@ EGIT_COMMIT="v${PV}"
 #endif
 
 RDEPEND="dev-libs/openssl:=
-	>=dev-libs/opensc-0.14
+	>=dev-libs/opensc-0.14[pcsc-lite]
```

Comment:
opensc 0.13 to 0.16 are all ~arch masked. unless 0.12 is good enough, might as well put 0.16 here. or what is the actual needed version?